icu4c/source/i18n/msgfmt.cpp

```diff
@@ -810,26 +810,31 @@ MessageFormat::getFormats(int32_t& cnt) const
     // method on this object.  We construct and resize an array
     // on demand that contains aliases to the subformats[i].format
     // pointers.
+
+    // Get total required capacity first (it's refreshed on each call).
+    int32_t total_capacity = 0;
```

Comment:
Please change this to totalCapacity (camelCase).

Reply:
done